daffodil-cli/build.sbt

```diff
@@ -26,10 +31,9 @@ enablePlugins(RpmPlugin)
 executableScriptName := "daffodil"
 
 packageName in Universal := "apache-daffodil-" + version.value + "-incubating-bin" //tarball name
-
 packageName in Linux := executableScriptName.value
-
 packageName in Rpm := "apache-" + executableScriptName.value
+packageName in Windows := executableScriptName.value
```

Comment:
I played with this a bit and found that 'packageName in Windows' changes and controls *nothing*. I made the recommended change and it had *no* effect on the behavior of the resulting MSI file. Since it is more consistent with the other stuff I'm leaving that change in.

Reply:
Looks like this causes some odd XML Id fields to be generated. It doesn't create anything that crashes, but the Id fields look really strange when you do this.

Reply:
If it doesn't have an affet and is only used in XML Id fields, let's keep as ``executableScriptName.value``. No need to make the generated XML uglier than it already is.

Reply:
Okay. I'll back that change out.